File: src/main/java/com/wxl/dyttcrawler/web/dto/search/DyttSimpleMovie.java
package com.wxl.dyttcrawler.web.dto.search;

import lombok.Data;

import java.io.Serializable;
import java.util.List;

/**
 * Create by wuxingle on 2020/7/17
 * 电影简单对象
 */
@Data
public class DyttSimpleMovie implements Serializable {

    private static final long serialVersionUID = -2563172955833336876L;

    private String id;

    // 标题
    private String title;

    // 地址链接
    private String url;

    // 封面图片地址
    private String picUrl;

    // 片名
    private String name;

    // 年代
    private Integer year;

    // 产地
    private List<String> originPlace;

    // 类别
    private List<String> category;

    // 豆瓣评分
    private Double score;

    // 标签
    private List<String> tags;

}

File: src/main/kotlin/com/wxl/dyttcrawler/web/dto/search/DyttSimpleMovie.kt
package com.wxl.dyttcrawler.web.dto.search

/**
 * Create by wuxingle on 2021/10/13
 * 电影简单对象
 */
data class DyttSimpleMovie(
    var id: String? = null,

    // 标题
    var title: String? = null,

    // 地址链接
    var url: String? = null,

    // 封面图片地址
    var picUrl: String? = null,

    // 片名
    var name: String? = null,

    // 年代
    var year: Int? = null,

    // 产地
    var originPlace: List<String>? = null,

    // 类别
    var category: List<String>? = null,

    // 豆瓣评分
    var score: Double? = null,

    // 标签
    var tags: List<String>? = null
)
